README.md

```diff
@@ -16,9 +16,22 @@ feel free to clone or copy any pieces that may be helpful.
 
 ## Getting A Test Account Up and Running
 
-You can use the demo wallet to test Hosted Deposit and Withdrawal (SEP-24) and Cross-Border Payments (SEP-31) with any home domain that has a Stellar Info File (also known as SEP-1, or a stellar.toml file). The instructions below are for demo-ing standard integrations supported by Stellar test server, testanchor.stellar.org. For both integrations, the logs to the right of the screen will show every network call. 
+You can use the demo wallet to test Regulated Assets ([SEP-8]), Hosted Deposit and Withdrawal ([SEP-24]) and Cross-Border Payments ([SEP-31]) with any home domain that has a Stellar Info File (also known as [SEP-1], or a stellar.toml file). The instructions below are for demo-ing standard integrations supported by Stellar test server, testanchor.stellar.org or by the [SEP-8] reference server, sep8-server.dev.stellar.org. For these integrations, the logs to the right of the screen will show every network call.
+
+### Demo-ing a Regulated Asset Payment ([SEP-8])
 
-### Demo-ing a Deposit on Testnet with Hosted Deposit and Withdrawal (SEP-24)
+1. Click "Generate keypair", and then click "Create account" - this will create a balance of 10,000 XLM.
+2. Click “Add asset” and add `MYASSET` with the anchor home domain `sep8-server.dev.stellar.org`.
+3. Click “Add trustline” - this will allow you to hold MYASSET.
+4. Click in the "Copy" link at the right of your public key and use that value to get some unities of MYASSET using the link `https://sep8-server.dev.stellar.org/friendbot?addr=<paste_your_address_here>`.
+5. Select "SEP-8 Send" from the dropdown for MYASSET and click "Start" in the modal.
+6. In the "destination" field, input an address that also has a trustline to MYASSET. You can use this destination address for it if you don't have one: `GBID36ML6VVNPIF6SQATQW5QIBREQAVEHQIUMWDLQCSVIYX2IJGK4KPP`.
```

Comment:
Will we remember to update this address when the testnet resets?

Reply:
Good point! I'll remove the reference to that address so we don't risk becoming outdated.

Reply:
Removed the sentence about the existing address in https://github.com/stellar/stellar-demo-wallet/pull/170/commits/6bb496a8fc971419c445d8277efa57ca51cb3d1c.